Add missing `i` key mapping

iarwain opened the pull request:
All is in the title. =)

ypujante commented:
How did I miss that???? Thanks for the PR!

iarwain commented:
My pleasure, thank you for the library in the first place!

ypujante commented:
I guess I never noticed and I just verified that my other project [WebGPU Shader Toy](https://pongasoft.github.io/webgpu-shader-toy/) actually works with the letter 'i' when entering it in the editor... I think it is the GLFW callback that doesn't work and Dear ImGui does not use the callback which is why it works..

ypujante commented on tools/create_keyboard_mapping.py:
did you rerun the tool to generate the mapping or just updated the mapping? The reason why I am asking is that the tool computes the [collision free hash](https://github.com/pongasoft/emscripten-glfw/blob/8b00c25413c9a721496043e336faceb7b14a68c4/src/cpp/emscripten/glfw3/KeyboardMapping.h#L198)

and I just want to make sure that it is still collision free...

iarwain replied:
I ran the tool to get the hash, however I only updated the line for `i`.
I assumed other hashes wouldn't have changed.

iarwain commented:
> I guess I never noticed and I just verified that my other project [WebGPU Shader Toy](https://pongasoft.github.io/webgpu-shader-toy/) actually works with the letter 'i' when entering it in the editor... I think it is the GLFW callback that doesn't work and Dear ImGui does not use the callback which is why it works..

I only noticed today, after months of using it, due to some cheat codes (`iddqd`-style) I added to a little test program.
The codes use key names to support different layouts. I wouldn't have noticed it otherwise.

ypujante commented:
@iarwain Thanks again for reporting and providing the PR. I merged it, reran the tool (which did not change the hash code but did change the `0X` into `0x` which I kept for consistency) and released it. 

I have also submitted a [PR into Emscripten](https://github.com/emscripten-core/emscripten/pull/23832) for the contrib port. So hopefully the changes will be widely available in the next Emscripten release.

You probably already know that but in case you do not, you don't have to wait for Emscripten to be released to start using it right away. Check the [documentation](https://github.com/pongasoft/emscripten-glfw?tab=readme-ov-file#note-about-availability-in-emscripten) here for details.

iarwain commented:
Thank you for the Emscripten PR submission and my bad for the `0X` capitalization: when I ran the tool, it did not update the file directly (I probably made a mistake there), but given the hash was in the terminal output, I simply copied it from there. However, it was lower case, so I updated it, making the mistake that affected `0x` as well in the process.

I'll check the documentation for using a specific version from `contrib`, thank you.
Until last night I was using a local clone I modified to support `pthread` as, when I first started using your library, you didn't have it supported yet.
